src/referencePackages/src/system.io.pipelines/5.0.1/System.IO.Pipelines.5.0.1.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.0;netstandard2.0</TargetFrameworks>
```

Comment:
This should be regenerated after https://github.com/dotnet/source-build-reference-packages/pull/1228 is merged.

Reply:
Regenerated in https://github.com/dotnet/source-build-reference-packages/pull/1227/commits/9d756e26e7e786b4e209ac8796dd51133d66da48